components/LikeCount.tsx

```diff
@@ -0,0 +1,34 @@
+import { useState } from 'react';
+
+function LikeCount({
+  type,
+  likeCount,
+}: {
+  type: 'readOnly' | 'interactive';
+  likeCount: number;
+}) {
+  const [isCheck, setIsCheck] = useState(false);
+  const [currentCount, setCurrentCount] = useState(likeCount);
+
+  const handleClick = () => {
+    setIsCheck(!isCheck);
+    setCurrentCount(isCheck ? currentCount - 1 : currentCount + 1);
+  };
+
+  return (
+    <>
+      <button
+        className="text-14 text-t-disabled mo:text-12"
+        onClick={handleClick}
+        disabled={type === 'readOnly'}
+      >
+        {!isCheck ? '♡' : '♥︎'}
```

Comment:
이거 그냥 이모티콘으로 때려넣으셨네요 ㅋㅋ 나중에 수정하실 예정이 있으신걸까요? 그냥 작업하시는김에 svgr로 넣어주시는게 더 좋을것같아욤 ㅎ.ㅎ
svg 파일 내부에 `inherit` 넣어주시는거 잊지마시구요!
```suggestion
        {!isCheck ? <ICON_HEART fill='none' stroke='var(--icon-primary)'/> : <ICON_HEART fill='var(--s-danger)' stroke='none'>}
```

Reply:
이건 급하게 작업을 하느라 깊게 생각을 안 해봤지만 아무래도 맥에서는 텍스트로 넣었을 때 괜찮게 보일지 몰라도 윈도우에서는 또 다르게 보일 수 있기에 정화님께서 알려주신 방법을 적용하는게 좋을 것 같습니다! 
감사합니다👍